logging/gokit.go

```diff
@@ -9,8 +9,11 @@ import (
 )
 
 // NewGoKit creates a new Interface backed by a GoKit logger
-func NewGoKit(l Level) Interface {
+func NewGoKit(l Level, format string) Interface {
 	logger := log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	if format == "json" {
+		logger = log.NewJSONLogger(log.NewSyncWriter(os.Stderr))
```

Comment:
Seems kinda wasteful to create and immediately discard the other one; suggest `if/else`.